src/k8s/pkg/component/component.go

```diff
@@ -83,7 +84,7 @@ func NewManager(snap snap.Snap) (*helmClient, error) {
 }
 
 // Enable enables a specified component.
-func (h *helmClient) Enable(name string) error {
+func (h *helmClient) Enable(name string, values map[string]any) error {
```

Comment:
yes, see `helmClient` does not implement the `ComponentManager` with this. A common approach to ensure we don't break the interface is to add the following at the end of the file, which ensures that a `*helmClient` can be assigned to a `ComponentManager` interface:

```go
var _ ComponentManager = &helmClient{}
```